### PR TITLE
CON-1810-MARKDOWN-drawing-raid-for-Java-piscine

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**
docs(Jart):add the audit of jart raid

> Before starting, please choose the relevant pull request **Labels**, **Reviewers**, and **Assignees**

### Why?

> Clearly state the reason for this change. What problem is it solving or feature is it adding?

### Solution Overview

> Provide an overview of the solution implemented in this pull request. This should be a high level overview without getting into technical details. If applicable, include screenshots of UI or use GitHub compliant [`mermaid`](https://mermaid.live/) graphs to visually represent the solution.

### Implementation Details

> Explain the details of the implementation and the reasoning behind it. What alternative approaches were considered and why was this approach chosen?

### CODE REVIEW

File: subjects/java/raids/Jart/README.md
Comment (zanninso) [inline subjects/java/raids/Jart/README.md:1]: the interface Displayable its not used here at all you can remove it while the image.save used to result the image or use display from Displayable interface to produce the image and require for the student to produce the image by opening a file and writing the pixels there you can use the easiest image format for not make it hard

File: subjects/java/raids/Jart/audit/README.md
Comment (zanninso) [inline subjects/java/raids/Jart/audit/README.md:1]: ask the students about the interfaces and if it's well used and implemented 
you can ask also if the student use a default implementation of in the interfaces it self

**Review by zanninso — CHANGES_REQUESTED** [review]
please resolve the added conversations

File: subjects/java/raids/Jart/README.md
Comment (Hamzaelkhatri) [reply subjects/java/raids/Jart/README.md:1]: The Displayable interface is implemented in the Image class. the method `save` for example is to get it from the Displayable interface

**Review by zanninso — CHANGES_REQUESTED** [review]

File: subjects/java/raids/Jart/README.md
Comment (Hamzaelkhatri) [reply subjects/java/raids/Jart/README.md:1]: from the example, the student should understand that the `Image` class implements `Displayable`. and Displayable contains two functions `display` and  `save`

```java
interface Displayable {
    void display(int x, int y, Color color);

    void save(String string);
}
```
display function used inside `Drawable` as a parameter of the `draw` function 

```java
interface Drawable {
    void draw(Displayable displayable);
   //...
}
```

> I add the `save`  function in the subject in the part of  `Displayable` interface to make it more clear.

Comment (zanninso) [reply subjects/java/raids/Jart/README.md:1]: there is no details of the interfaces methods signature in the subject and this make it harder even for me to understand, and in this way we keep a gap where students can do a lot of mistakes you have to specify the signatures of the interfaces methodes, like in the example same the object of the image into a variable of type displayable this way can give a hint to the students about the signatures and parametres types.
to recap here is what you have to change:
- add the signature of the display method that will take the corrdinates`x, y` and a color
- change the image variable type to `Displayable `

File: subjects/java/raids/Jart/audit/README.md
Comment (Hamzaelkhatri) [reply subjects/java/raids/Jart/audit/README.md:1]: Done 👍🏻

**Review by zanninso — APPROVED** [review]
LGTM
